ui_config/tenant/YJSG-Indore/routes/memberRegistration.json

```
@@ -668,7 +668,9 @@
 				},
 				"remark": {
 					"title": "Remark",
-					"type": "string"
+					"type": "string",
+					"enum": ["yes", "no", "other"],
```

Can we add some generic values for now
```suggestion
					"enum": ["Not picking up call", "Level needs to be updated", "Classroom not assigned", "ID card lost",  "Other"],
```